systemimage/device.py

```diff
@@ -38,13 +38,17 @@ class SystemProperty(BaseDevice):
 
     def get_device(self):
         log = logging.getLogger('systemimage')
+        stdout = ''
         try:
-            stdout = check_output(
-                'getprop ro.product.device'.split(), universal_newlines=True)
-        except CalledProcessError as error:
-            log.exception('getprop exit status: {}'.format(error.returncode))
-            return '?'
-        except FileNotFoundError as error:
-            log.exception('getprop command not found')
-            return '?'
-        return stdout.strip()
+            stdout = check_output('getprop ro.product.device'.split(), universal_newlines=True).strip()
+        except:
```

Comment:
Please do not use blanket exceptions. Instead this should be `except CalledProcessError:` to only catch the failure to run getprop.